modules/ROOT/pages/twilio/twilio-connector.adoc

```diff
@@ -148,9 +148,9 @@ For more information on each format, see the See Also section at the end of this
 
 == Use Case: Studio
 
-In the following example, a Mule application sends a message to a phone number, and then redacts it.
+In the following example, a Mule application sends a message to a phone number, and then redacts it. This example uses variables for field values. You can provide the values for each variable in a properties file. To use a properties file, place the file at `/src/main/resources` and use the tag `<configuration-properties>` to point to the file. For more information, see xref:mule-runtime::configuring-properties.adoc#properties-files[Properties Files].
 
-. Create a new Mule application and add the following properties to the mule-app.properties file:
+. Create a new Mule application and add the following properties to the properties file:
 +
 [%header,cols="30s,70a"]
 |===
```

Comment:
HIDDEN TEXT
Edited; please compare
(mostly for lowercase and no periods....)

|Property |Description
|accountSid |Your account SID
|authToken |Your authentication token
|fromNumber |The phone number from which the SMS is to be sent (the phone number you received from Twilio)
|===
+
. Add an empty flow, drag an HTTP connector to the inbound part of the flow, and set its path to `/send/{toNumber}`.
. Drag a transform message to the flow and prepare the input for the Twilio connector:
+
[source,dataweave,linenums]
----
%dw 2.0
output application/java
---